app/livechat/server/hooks/sendToFacebook.js

```diff
@@ -33,5 +34,5 @@ callbacks.add('afterSaveMessage', function(message, room) {
 		text: message.msg,
 	});
 
-	return message;
+	return normalizeMessageAttachments(message);
```

Comment:
Do you mind of checking the `message.file` before returning? I think we should always return the `message` instead of the `normalizeMessageAttachments`.